Applications/Cxx/gdcmanon.cxx

```diff
@@ -324,6 +326,7 @@ int main(int argc, char *argv[])
     static struct option long_options[] = {
         {"input", required_argument, NULL, 'i'},                 // i
         {"output", required_argument, NULL, 'o'},                // o
+        {"json", required_argument, NULL, 'j'},                  // j
```

Comment:
I added the helpful //j comment to match the other helpful comments already present